src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/main.js

```diff
@@ -11,8 +11,9 @@ define([
     'sulucontent/model/content',
     'sulucontent/components/content/preview/main',
     'sulucontent/components/copy-locale-overlay/main',
+    'sulusecurity/services/security-checker',
     'config'
-], function(Content, Preview, CopyLocale, Config) {
+], function(Content, Preview, CopyLocale, SecurityChecker, Config) {
```

Comment:
I've never seen an injection starting with a lower letter, and actually almost none of our modules are constructors. I think in this list `Preview` is the only one being a constructor.